terraform-plans/templates/github/snap_release.yaml.tftpl

```diff
@@ -10,15 +10,19 @@ on:
     branches: [ ${branch} ]
   release:
     types: [ published ]
+  workflow_dispatch:
 
 jobs:
+{{ if include_check_job }}
   check:
     uses: ./.github/workflows/check.yaml
     secrets: inherit
-
+{{ endif }}
   release:
     runs-on: ubuntu-latest
+{{ if include_check_job }}
     needs: check
+{{ endif }}
```

Comment:
I would prefer to avoid conditionals in templates.  Can we avoid this by adding a check.yaml workflow to charmed-openstack-exporter-snap?  That would be a win-win then.  It only needs to verify that building the snap succeeds.

Reply:
Ok. I think the workflow for the charmed-openstack-exporter is a corner case and not sure if worth adding a new template for it. So I'll try to keep simple and kust have the `promote` and `CODEOWNERS`. If we see a need in the future, we can open a new PR

Reply:
I don't think it's a corner case.  I think it's just as useful to have a check workflow as our other projects.  Please see my [other comment](https://github.com/canonical/solutions-engineering-automation/pull/41#discussion_r1714375910) for useful checks to include in the check workflow.

Reply:
@gabrielcocenza thanks for addressing the other comments, it looks fine to me now.  I still argue that we should have a test workflow, so I opened https://github.com/canonical/charmed-openstack-exporter-snap/issues/5 to track it.